rest_framework_simplejwt/authentication.py

```diff
@@ -164,6 +165,37 @@ def get_user(self, validated_token: Token) -> AuthUser:
         return api_settings.TOKEN_USER_CLASS(validated_token)
 
 
+class JWTInactiveUserAuthentication(JWTAuthentication):
```

Comment:
This class shouldn't be needed given the new settings

Reply:
oops, i missed it. should be good now